### PR TITLE
Support react16

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

**Comment by openjck** [comment]
I'll took a look tomorrow. Thanks, William!

File: package.json [hunk 1]
```diff
@@ -13,13 +13,12 @@
...
-  "dependencies": {
-    "classnames": "^2.1.2",
-    "metrics-graphics": "^2.12.0"
-  },
+  "files": [
+    "build"
+  ],
   "peerDependencies": {
     "metrics-graphics": "^2.11.0",
+    "prop-types": "^15.6.0",
```
Comment (openjck) [inline package.json]: I've never quite understood `peerDependencies`, so this is probably just be my ignorance asking, but does this need to be a `peerDependency`?

Comment (wlach) [reply package.json]: Yes, this is a pretty good primer: 

* https://nodejs.org/en/blog/npm/peer-dependencies/
* https://stackoverflow.com/a/34645112

Basically this avoids having prop-types being specified twice in the dependency graph for the project.

**Comment by ysokolovsky** [comment]
@wlach hi! When npm package will be updated?

**Comment by wlach** [comment]
@ysokolovsky just published version 1.0.2 now, seems to work ok!

**Pull request merged**